src/inet/UDPEndPointImplLwIP.cpp

```diff
@@ -140,14 +144,15 @@ CHIP_ERROR UDPEndPointImplLwIP::LwIPBindInterface(struct udp_pcb * aUDP, Interfa
         }
     }
 
-    udp_bind_netif(aUDP, netifp);
+    RunOnTCPIP([aUDP, netifp]() {
+        udp_bind_netif(aUDP, netifp);
+        return ERR_OK;
+    });
     return CHIP_NO_ERROR;
 }
 
 InterfaceId UDPEndPointImplLwIP::GetBoundInterface() const
 {
-    ScopedLwIPLock lwipLock;
```

Comment:
Why is synchronization here no longer needed?

Reply:
RunOnTCPIP is the synchronization.

Reply:
But this function is not calling RunOnTCPIP anywhere.

Reply:
yes, my bad, should have wrapped `netif_get_by_index`. done.